libraries/botbuilder-dialogs/tests/test_choice_prompt.py

```diff
@@ -318,11 +315,8 @@ def test_choice_prompt_with_none_id_should_fail(self):
         step1 = await adapter.send(
             Activity(type=ActivityTypes.message, text="Hello", locale=Culture.Spanish)
         )
-        # TODO ChoiceFactory.inline() is broken, where it only uses hard-coded English locale.
-        # commented out the CORRECT assertion below, until .inline() is fixed to use proper locale
-        # step2 = await step1.assert_reply('Please choose a color. (1) red, (2) green, o (3) blue')
         step2 = await step1.assert_reply(
-            "Please choose a color. (1) red, (2) green, or (3) blue"
+            'Please choose a color. (1) red, (2) green, o (3) blue'
```

Comment:
Because we're adopting black code formatter soon, could you change the single quote marks for " (double) for all the strings please?

Reply:
Oh good! I like single quotes better for strings anyways :)

Reply:
Done! Changed double quotes to single quotes in ChoicePrompt & Prompt classes, and also ChoicePrompt unit tests